This mod seems to be incompatible with latest AzerothCore build

Hello!
I just wanted to let you know this mod seems to be incompatible with the latest (April 2021) release of AzerothCore, as the build process fails.

Please see below the compile error:
```
[ 99%] Building CXX object src/server/scripts/CMakeFiles/scripts.dir/__/__/__/modules/mod-instance-reset/src/InstanceReset.cpp.o
/opt/azerothcore-wotlk/modules/mod-instance-reset/src/InstanceReset.cpp:48:111: fatal error: no member named 'GetGUIDLow' in 'Player'
                BoundInstancesMap const& m_boundInstances = sInstanceSaveMgr->PlayerGetBoundInstances(player->GetGUIDLow(), Difficulty(i));
                                                                                                      ~~~~~~  ^
```

Thanks for your work :-)

--- a/src/InstanceReset.cpp
+++ b/src/InstanceReset.cpp
@@ -45,12 +45,12 @@ public:
                 diff = MAX_DIFFICULTY;
             for (uint8 i = 0; i < diff; ++i)
             {
-                BoundInstancesMap const& m_boundInstances = sInstanceSaveMgr->PlayerGetBoundInstances(player->GetGUIDLow(), Difficulty(i));
+                BoundInstancesMap const& m_boundInstances = sInstanceSaveMgr->PlayerGetBoundInstances(player->GetGUID(), Difficulty(i));
                 for (BoundInstancesMap::const_iterator itr = m_boundInstances.begin(); itr != m_boundInstances.end();)
                 {
                     if (itr->first != player->GetMapId())
                     {
-                        sInstanceSaveMgr->PlayerUnbindInstance(player->GetGUIDLow(), itr->first, Difficulty(i), true, player);
+                        sInstanceSaveMgr->PlayerUnbindInstance(player->GetGUID(), itr->first, Difficulty(i), true, player);
                         itr = m_boundInstances.begin();
                     }
                     else

